txs/cache.go

```diff
@@ -261,9 +246,26 @@ func (ac *accountCache) addBatch(logger log.Log, nonce2TXs map[uint64][]*NanoTX,
 
 	ac.moreInDB = len(sortedNonce) > len(added)
 	if len(added) > 0 {
-		logger.With().Debug("added batch to account pool", log.Array("batch", nonceMarshaller(added)))
+		logger.Debug("added batch to account pool",
+			zap.Array("batch", zapcore.ArrayMarshalerFunc(func(encoder zapcore.ArrayEncoder) error {
+				sort.Slice(added, func(i, j int) bool { return added[i] < added[j] })
```

Comment:
Isn't it equivalent to
```suggestion
				slices.Sort(added)
```

Reply:
True, I'll update the code.